src/nixupdate/nix_outdated.py

```diff
@@ -15,7 +15,8 @@
 from argparse import ArgumentParser
 from tabulate import tabulate
 from sbomnix.sbomdb import SbomDb
-from sbomnix.utils import (
+import repology.repology_cli
+from common.utils import (
     LOG,
     LOG_SPAM,
     set_log_verbosity,
```

Comment:
do we still use exec_cmd imported here?

Reply:
exec_cmd is still needed to run `nix-visualize`: https://github.com/tiiuae/sbomnix/blob/dea08ff12a2c337a31addfc870ec1fd10d66632f/src/nixupdate/nix_outdated.py#L107